app/services/ApiService.ts

```diff
@@ -64,6 +66,10 @@ export const signUp = (user: NewUser): Promise<string> => {
   return fetchFrom<AuthResponse>(`${AuthUrl}/signUp`, "POST", user).then((resp) => resp.accessToken);
 };
 
+export const getSignMessage = (address: string): Promise<ApiSignMessage> => {
+  return fetchFrom<ApiSignMessage>(`${AuthUrl}/signMessage?address=${address}`);
+};
+
```

Comment:
FYI
I added a (url) queryParam option in my implementation version, but not yet really needed now (just so you know where to look/copy if it's needed in the future 😉)
https://github.com/DFXswiss/wallet/pull/21/files#diff-0e865de905705adfe88bf0022e860df006ed1df41266d705f8597bcd2aa794c0